content/guides/rom/v5.0/core-concepts/changesets.md

```diff
@@ -155,7 +155,9 @@ task
 ```
 
 > [!NOTE]
+>
 > #### Association name
+>
 > Notice that `associate` method can accept a rom struct and it will try to infer association name from it. If this fails because you have an aliased association then pass association name explicitly as the second argument, ie: `associate(user, :author)`.
```

Comment:
I think in most of my work porting the rom docs over, I got rid of most of these "headings inside alert banners" structures. We could probably do it here too.

Reply:
Have removed for now!